### ISSUE TITLE
<T> generics and Node.js streams

### ISSUE THREAD

**Issue by ORESoftware** [Microsoft/TypeScript#14462]
I see this warning and not sure why I get the warning - I assume it's a TS thing, but not 100% certain. 

<img width="975" alt="screenshot 2017-03-04 23 09 31" src="https://cloud.githubusercontent.com/assets/11139560/23585299/df7b8a06-012f-11e7-9fcd-90c279faf15e.png">

Any explanation would be good to know, thanks

**Comment by aluanhaddad** [comment]
A bit hard to parse this as a partial screenshot from IntelliJ IDEA/WebStorm. Could you post the program text?

**Comment by RyanCavanaugh** [comment]
Please provide self-contained repros, not screenshots deep in your codebase. This is not a support forum.
